src/DynamoCoreWpf/ViewModels/PackageManager/PackageViewModel.cs

```diff
@@ -233,24 +256,125 @@ private void Uninstall()
             }
             finally
             {
-                RaisePropertyChanged(nameof(PackageLoadStateTooltip));
-                RaisePropertyChanged(nameof(PackageLoadStateText));
+                NotifyLoadStatePropertyChanged();
             }
         }
 
         private bool CanUninstall()
         {
             if (!Model.InUse(dynamoViewModel.Model) || Model.LoadedAssemblies.Any())
             {
-                return Model.BuiltInPackage ? 
+                return Model.BuiltInPackage ?
                     Model.LoadState.State != PackageLoadState.StateTypes.Unloaded &&
                     Model.LoadState.ScheduledState != PackageLoadState.ScheduledTypes.ScheduledForUnload :
                     Model.LoadState.ScheduledState != PackageLoadState.ScheduledTypes.ScheduledForDeletion;
             }
             return false;
         }
 
-        private void GoToRootDirectory()
+        // Loads a built-in package that was previously set as Unloaded
+        private void Load()
+        {
+            var dynModel = dynamoViewModel.Model;
+            var packageLoader = dynModel.GetPackageManagerExtension().PackageLoader;
+            var conflicts = packageLoader.LocalPackages.ToList().Where(x => x.Name == Model.Name && x != Model);
+            bool hasConflictsWithLoadedAssemblies = conflicts.Any(x => x.InUse(dynamoViewModel.Model));
+
+            if (conflicts.Any())
+            {
+                string conflictsMsg = string.Join(",", conflicts.Select(x => x.Name + " " + x.VersionName));
+
+                string message = hasConflictsWithLoadedAssemblies ? 
+                        string.Format(Resources.MessageLoadBuiltInPackageWithRestart,
+                        conflictsMsg, Model.Name + " " + Model.VersionName) 
+                        :
+                        string.Format(Resources.MessageLoadBuiltInPackage,
+                        conflictsMsg, Model.Name + " " + Model.VersionName);
+
+                var dialogResult = MessageBoxService.Show(message,
+                    Resources.CannotDownloadPackageMessageBoxTitle,
+                        MessageBoxButton.OKCancel,
+                        MessageBoxImage.Exclamation);
+
+                if (dialogResult == MessageBoxResult.Cancel) return;
+            }
+
+            try
+            {
```

Comment:
It's hard for me to tell, does anything in this try catch ever result in an error state?

Reply:
I do not think it does.
I think `UninstallCore` can throw if the files on disk are write protected..
Not sure if we should continue or exit the "Load" command immediately in that case...